### PR TITLE
feat(): Add dry option to run without committing changes

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
Sometimes you'd prefer tools to just tell you what's wrong, rather than automatically making changes and commits in your repository

### CODE REVIEW

**Comment by billyvg** [comment]
@ThisIsMissEm This seems reasonable -- I'm wondering if this action (with dry run) provide value over running eslint by itself in a workflow?

File: action.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ inputs:
   GITHUB_TOKEN:
     description: 'GitHub token'
     required: true
+  dry:
+    description: Just runs eslint, doesn't commit any changes
+    required: false
+    default: false
```
Comment (billyvg) [inline action.yml:14]: Will this get cast as a string causing https://github.com/getsentry/action-eslint-fix/pull/7/files#diff-8cfead41d88ad47d44509a8ab0a109adR68 to always be true?

**Comment by ThisIsMissEm** [comment]
@billyvg this went from "seems reasonable" to "closed"? sorry, I've been quite busy and hadn't time to come back to this, if you wanted me to update it, you could've given me a polite nudge.

**Comment by billyvg** [comment]
@ThisIsMissEm Ah, I updated the main branch from `master` to `main` and it automatically closed it out

**Comment by ThisIsMissEm** [comment]
@billyvg ah, okay, have reopened it against `main`

**Pull request closed without merging**